app/scripts/lib/setupSentry.js

```diff
@@ -2,13 +2,15 @@ import { createModuleLogger, createProjectLogger } from '@metamask/utils';
 import * as Sentry from '@sentry/browser';
 import { logger } from '@sentry/utils';
 import browser from 'webextension-polyfill';
+import { v4 as uuidv4 } from 'uuid';
 import { isManifestV3 } from '../../../shared/modules/mv3.utils';
 import { getManifestFlags } from '../../../shared/lib/manifestFlags';
 import extractEthjsErrorMessage from './extractEthjsErrorMessage';
 import { filterEvents } from './sentry-filter-events';
 
 const projectLogger = createProjectLogger('sentry');
 let installType = 'unknown';
+let sentryUserId = null;
```

Comment:
Q: is setting a variable like this at global level a good practice in extension?

Reply:
Could use insight from @davibroc or @MajorLift ❤️